website/src/components/SearchPage/DownloadDialog/generateDownloadUrl.ts

```diff
@@ -38,31 +39,48 @@ export const generateDownloadUrl = (
     if (option.compression !== undefined) {
         params.set('compression', option.compression);
     }
-    if (lapisSearchParameters.accession !== undefined) {
-        for (const accession of lapisSearchParameters.accession) {
-            params.append('accession', accession);
-        }
-    }
 
-    const mutationKeys = ['nucleotideMutations', 'aminoAcidMutations', 'nucleotideInsertions', 'aminoAcidInsertions'];
+    switch (downloadParameters.type) {
+        case 'filter':
+            const lsps = downloadParameters.lapisSearchParameters;
```

Comment:
Our style is mostly not to abbreviate variable names
```suggestion
            const lapisSearchParameters = downloadParameters.lapisSearchParameters;
```

Reply:
https://github.com/loculus-project/loculus/pull/2917/commits/b0431edffde9765810616afeb528103876f2884f